src/back/TicketManager/src/Core/TicketManager.Core.Domain/Events/SeatReservation.cs

```diff
@@ -0,0 +1,7 @@
+namespace TicketManager.Core.Domain.Events;
+
+public record SeatReservation(Guid SectorId, Guid UserId, int ReservedSeatNumber)
+{
+    public Guid Id = Guid.NewGuid();
+    public DateTime CreationDate = DateTime.UtcNow;
+}
```

Comment:
It's wrong with so many things that it's hard to comment.

1. Public variables.
2. You sent new `Id` and `CreationDate` every single time you create instance and that mean that also when ef create instance it set new values to those fields.

Reply:
I don't really understand you problem. Here you have `SeatReservation` when it is the only item and then one when second item was added. As you can see both `Id` and `CreationDate` are initialized on creation not every time they are pulled from DB
![image](https://user-images.githubusercontent.com/43820607/232612397-6ba9cf14-492c-43d1-9964-cde3931fb745.png)
![image](https://user-images.githubusercontent.com/43820607/232612420-89f1e3ff-b3ef-4c0a-89c7-4f8e92bb5845.png)

Reply:
And public variable is exactly the same thing as you did with `Sector` when it was yet a `record`

Reply:
Records generates init only properties by default. PROPERTIES not VARIABLES.

Reply:
But with setting values you're right, my bad.